packages/expo-asset/package.json

```
@@ -15,7 +15,9 @@
     "test": "expo-module test",
     "prepare": "expo-module prepare",
     "prepublishOnly": "expo-module prepublishOnly",
-    "expo-module": "expo-module"
+    "expo-module": "expo-module",
+    "open:ios": "open -a \"Xcode\" example/ios",
+    "open:android": "open -a \"Android Studio\" example/android"
```

It doesn't have the example app
```suggestion
    "expo-module": "expo-module"
```